standalone/ansible/roles/load_balancing/tasks/main.yml

```diff
@@ -1,22 +1,22 @@
 # load balancing tasks
 ---
-- name: Load Balancing | Install HAProxy
+- name: install HAProxy
```

Comment:
Are we abandoning our pretty message formatting? Is this message displayed anywhere? If so, I _really_ like the older version.

Reply:
This is mostly for the ansible logs, where they are already displayed with the namespace like so:
```
load_balancing: Load Balancing | Install HAProxy
```

Seemed unnecessary. The messages look like this now:
```
load_balancing: install HAProxy
```

Reply:
Ah, each line has a namespace attached. Nice. 👍 v